internal/exec/utils.go

```diff
@@ -152,10 +148,33 @@ func processConfigAndStacks(componentType string, cmd *cobra.Command, args []str
 	configAndStacksInfo.DeployRunInit = argsAndFlagsInfo.DeployRunInit
 	configAndStacksInfo.AutoGenerateBackendFile = argsAndFlagsInfo.AutoGenerateBackendFile
 	configAndStacksInfo.UseTerraformPlan = argsAndFlagsInfo.UseTerraformPlan
+	configAndStacksInfo.NeedHelp = argsAndFlagsInfo.NeedHelp
+
+	// Check if `-h` or `--help` flags are specified
+	if argsAndFlagsInfo.NeedHelp == true {
```

Comment:
```suggestion
	if argsAndFlagsInfo.NeedHelp {
```

Reply:
yea, nice find.
it's opinionated since Go is a very explicit language